week-4/F-object-keys/README.md

```diff
@@ -1,5 +1,18 @@
 ### Goals
 
-- explain `Object.keys()` 
-- good time to mention how `Object` is just another JS object provided by the runtime that we can access anytime (similar to `console`)
-- continue with more advanced exercises that use `Object.keys` but also build up on the previous topics
\ No newline at end of file
+`Object.keys()` allows your to iterate over an object's keys, in the same way that a `for..in` loop would, and return an array of strings of your object's keys.
+
+Object is a type that has built-in methods attached to it - such as `.keys()` or `.values()`. You're all very familiar with `console.log()` - `Object` is helpfully provided to us.

```

Comment:
I dunno if `Object` is really best desribed as a type. I asked @rarmatei  and he said "`Object` is just another "object" available on the global `window` object", but that's probaby not much help here :). Also I don't think they've seen `for..in` or are familiar with the word "iterate" yet.